.github/CODEOWNERS

```diff
@@ -116,3 +116,6 @@ browser/net/brave_network_audit_allowed_lists.h @brave/sec-team
 # iOS
 ios/ @brave/ios
 build/ios/ @brave/ios
+
+# Android tests
+android/android_browser_tests.gni @wchen342
```

Comment:
can you create a group so there is more than one person who can approve?

Reply:
Created a new `android-tests-reviewers` team for reviewing this file.